src/hooks/use-list-behavior.js

```diff
@@ -0,0 +1,72 @@
+import { useState, useCallback } from 'react'
+
+/**
+ * Hook that returns bind props + accessibility props for
+ * "selectable" elements
+ */
+
+const resetStyles = {
+  outline: 'none'
+}
```

Comment:
If a hook is responsible for setting any styles it should do so via inline styles (and merge them with user-provided styles), IMO.

This works only because it happens to (now) be a Box-supported prop, but that was only recently added. This could be a gotcha easily where a css prop is used that Box does not support.